std_msgs/Header header

float64 gps_time
float32 geoid_height


